core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -216,6 +217,12 @@ public function configureFilesystem(FilesystemConfiguration $config): void
             ->addDefaultDbafs($filesStorageName, 'tl_files')
             ->addMethodCall('setDatabasePathPrefix', [$uploadPath]) // Backwards compatibility
         ;
+
+        // Backups
+        $config
+            ->mountLocalAdapter('var/backups', 'var/backups', 'backups')
+            ->addVirtualFilesystem('backups', 'var/backups')
```

Comment:
I wonder if we should make backups available directly under `backups` and just set the path of the mount to `var/backups` by default?

```suggestion
            ->mountLocalAdapter('var/backups', 'backups', 'backups')
            ->addVirtualFilesystem('backups', 'backups')
```

Reply:
Changed 👍